### PR TITLE
Feature/dynamic articles rework part 2

### PR DESCRIPTION
**Pull request by ArminWindlin**

### CODE REVIEW

File: src/views/article.vue [hunk 1]
```diff
@@ -1,69 +1,100 @@
...
+                    for (let key of subKeys) {
+                        key = key.trim();
+                        data = data[key] ? data[key] : 'INVALID_DATA_KEY';
+                    }
+                    parsedText += data;
+                    if (subFragment[1]) parsedText += subFragment[1];
+                } else parsedText += fragment;
             });
+            return parsedText;
+        },
```
Comment (Zahikusa) [inline src/views/article.vue:89]: Woah, nice one! At first when I saw you using the ``{{ data }}`` syntax in a json-file I thought: "How is this supposed to work?". Now I see how.

File: src/views/article.vue [hunk 2]
```diff
@@ -1,69 +1,100 @@
...
+        <div class="paragraph" v-for="(chapter, i) in article.chapters" :key="chapter.index">
             <!-- DIVIDER -->
             <img class="divider desktop-only" src="../assets/groups/vector-divider.svg" alt="">
             <img class="divider mobile-only" src="../assets/groups/divider-mobile.svg" alt="">
             <!-- PARAGRAPH TEXT AND IMAGE -->
             <div class="paragraph-content" :class="{'reverse-element': i%2 !== 0}">
-                <div class="image-box"
-                     :style="{ 'background-image': 'url(' + require('../assets/articles/' +  articleName + '/' + wikiArticles.chapters[i].image) + ')' }"
-                     :class="{'reverse-image': i%2 !== 0}">
+                <div class="image-box" v-if="article.chapters[i].image"
```
Comment (Zahikusa) [inline src/views/article.vue:22]: I think you could simplify the statement by using the iterator variable of the ``v-for`` above:

``v-if="article.chapters[i].image"`` -> v-if="chapter.image"
``v-if="article.chapters[i]"`` -> v-if="chapter"

File: src/views/wiki.vue [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    name: 'Wiki',
+    data() {
+        return {
+            structure: [
+                {
+                    name: 'Fields',
+                    folderName: 'fields',
+                    articles: [
+                        {name: 'Artifact', fileName: 'artifact'},
+                        {name: 'Control Point', fileName: 'control'},
```
Comment (Zahikusa) [inline src/views/wiki.vue:27]: Detail: We might better name this article ``Control Field`` due to the context here. (explaining all types of special fields)

**Review by Zahikusa — APPROVED** [review]
Looks great overall!

I have another task we might wanna add for the future: Implementing some kind of in-app-navigation-flow. For example a "back-button" would be cool. For now you can just use the arrow-button of the browser, but I think we could enhance the navigation-experience by providing an of navigation-element with additional information like "name of the article you would jump back to" etc.

**Review by MrMichael3 — APPROVED** [review]

**Pull request merged**